content/blog/what-happened-to-uuid-v2/index.md

```diff
@@ -0,0 +1,282 @@
+---
+{
+    title: "What Happened to UUIDv2?",
+    description: "You may have heard that "UUIDv2 is bad". Maybe you've never even heard of them and thought UUIDs went from v1 to v3. Why is that? What did UUIDv2 do wrong?",
+    published: '2023-01-15T21:52:59.284Z',
+    authors: ['crutchcorn'],
+    tags: ['computer science'],
+    attached: [],
+    license: 'cc-by-4',
+    series: "Explaining UUIDs",
+    order: 2
+}
+---
+
+If you read [my last post introducing UUIDs](/posts/what-are-uuids), you'll remember that I said:
+
+> UUIDv2 is _weird_.
+
+After this, [I left a very short (one or two sentences) explanation of what a UUIDv2 is](/posts/what-are-uuids#UUIDv2).
+
+Why would I do that [when UUIDv3 and UUIDv5 got four whole headers worth of explanation](/posts/what-are-uuids#UUIDv3and5)? What makes UUIDv2 different or "weird," as I put it?
+
+The answer is that while UUID _is_ a form of UUID by some definitions, it effectively is not in others.
+
+> What?!
+
+I know... It's a bit jarring. What's worse, even within the definitions we _do_ have, UUID is deeply flawed. Enough so that most are comfortable leaving it in the past.
```

Comment:
```suggestion
I know... It's a bit jarring. But, even within the definitions we have, UUID is deeply flawed, enough so that most are comfortable leaving it in the past.
```